packages/front-end/components/EventWebHooks/EventWebHookAddEditModal/EventWebHookAddEditModal.tsx

```diff
@@ -108,6 +139,20 @@ export const EventWebHookAddEditModal: FC<EventWebHookAddEditModalProps> = ({
         />
       </div>
 
+      <SelectField
+        label="Method"
+        value={form.watch("method")}
+        placeholder="Choose HTTP method"
+        options={eventWebHookMethods.map((method) => ({
+          label: method,
+          value: method,
+        }))}
+        onChange={(value: EventWebHookMethod) => {
+          form.setValue("method", value);
+          handleFormValidation();
+        }}
```

Comment:
fyi: `sort={false}` is needed if you want to preserve the order of options

Reply:
Applied, thanks